tests/executor/test_executor.py

```diff
@@ -42,6 +45,13 @@ def docs_with_tags(N):
     return da
 
 
+def clear_hubble():
+    client = hubble.Client(max_retries=None, jsonify=True)
+    art_list = client.list_artifacts()
+    for art in art_list['data']:
```

Comment:
we should pass the `name` for the artifact, otherwise, it will delete all artifacts under the specific Hubble account which is dangerous.

Reply:
We use a test account in unittest. I think it should clear all arts before starting a new test since we only clear arts at the beginning and the end of the test, it something wrong and the test stopped in the duration then the arts will not be cleared, so errors will pop in the next time of unittest. We couldn't predict how and where the test would stopped.